app/assets/javascripts/sidebar.js

```diff
@@ -0,0 +1,135 @@
+// patternfly grid-based sidebar resizer
+$(function() {
+  // Function to increment the Y in the col-XX-YY by add
+  var change_col = function(colstr, add) {
+    var arr = colstr.split('-');
+    var pos = arr.length - 1;
+    arr[pos] = parseInt(arr[pos], 10) + add;
+    return arr.join('-');
+  }
+
+  // Function to get the current column width
+  var get_col = function(colstr) {
+    var arr = colstr.split('-');
+    return parseInt(arr[arr.length - 1], 10);
+  }
+
+  // Function to resize the grid
+  var resize = function(event) {
+    var left = $(this).hasClass('resize-left');
+    var button = left ? $(this).next() : $(this);
+    var ajax = 2;
```

Comment:
Maybe a comment about the purpose of `ajax` variable? (I have no idea what it does.)